oxidation/client/src/components/SettingsModal.vue

```diff
@@ -0,0 +1,344 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS -->
+
+<template>
+  <ion-page class="modal">
+    <!-- top -->
+    <ion-header class="modal-header">
+      <ion-toolbar class="modal-header__toolbar">
+        <ion-title class="title-h2">
+          {{ $t('SettingsPage.pageTitle') }}
+        </ion-title>
+        <ion-buttons
+          slot="end"
+          class="closeBtn-container"
+        >
+          <ion-button
+            slot="icon-only"
+            @click="closeModal()"
+            class="closeBtn"
+          >
+            <ion-icon
+              :icon="close"
+              size="large"
+              class="closeBtn__icon"
+            />
+          </ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+    <!-- content -->
+    <ion-content class="modal-content">
+      <div class="menu">
+        <!-- menu list -->
+        <ion-radio-group
+          v-model="showTOS"
+          value="general"
+          class="menu-list"
+        >
+          <ion-radio
+            slot="start"
+            value="general"
+            class="menu-list__item"
+          >
+            <div class="item-container">
+              <ion-icon
+                :icon="cog"
+              />
+              <ion-text class="body">
+                {{ $t('SettingsPage.general') }}
+              </ion-text>
+            </div>
+          </ion-radio>
+          <ion-radio
+            slot="start"
+            value="advanced"
+            class="menu-list__item"
+          >
+            <div class="item-container">
+              <ion-icon
+                :icon="options"
+              />
+              <ion-text class="body">
+                {{ $t('SettingsPage.advanced') }}
+              </ion-text>
+            </div>
+          </ion-radio>
+        </ion-radio-group>
+        <!-- list item content -->
+        <div class="menu-item-content">
+          <!-- general -->
+          <div
+            v-if="showTOS != 'advanced'"
+            class="settings-general"
+          >
+            <ion-list class="settings-list">
+              <!-- synchro wifi -->
+              <settings-option
+                :title="$t('SettingsPage.synchroWifiOnly')"
+                :description="$t('SettingsPage.synchroWifiOnlyDescription')"
+                v-model="config.synchroWifiOnly"
+              />
```

Comment:
Does this setting is only relevant on `smartphone` ?

Reply:
The name is probably a bit misleading, I should have more to do with metered connection rather than wifi.

Reply:
As discussed with @Max-7, we can't handle this setting for now. So I think we should add the locales for better understanding and hide it for now.